browser/ui/brave_browser_command_controller.cc

```diff
@@ -182,6 +184,18 @@ void BraveBrowserCommandController::InitBraveCommandState() {
   UpdateCommandEnabled(IDC_APP_MENU_IPFS_OPEN_FILES, true);
 #endif
   UpdateCommandEnabled(IDC_BRAVE_BOOKMARK_BAR_SUBMENU, true);
+
+  UpdateCommandEnabled(IDC_TOGGLE_VERTICAL_TABS, true);
+  UpdateCommandEnabled(IDC_TOGGLE_VERTICAL_TABS_WINDOW_TITLE, true);
```

Comment:
I think we only enable these two commands when vertial tab flag is on?

Reply:
Oh wow, I've had this on by default for so long I thought the flag had been removed :laughing: